src/sirinternal.c

```diff
@@ -1062,8 +1062,17 @@ bool _sir_formattime(time_t now, char* buffer, const char* format) {
         return _sir_seterror(_SIR_E_INVALID);
 
     struct tm timebuf = {0};
+#if defined(__GNUC__) && !defined(__clang__) && \
+    !(defined(__OPEN64__) || defined(__OPENCC__))
+# pragma GCC diagnostic push
+# pragma GCC diagnostic ignored "-Wformat-nonliteral"
+#endif
     size_t fmttime    = strftime(buffer, SIR_MAXTIME, format,
```

Comment:
Same thing here. What if format is const char* const? Does that alleviate it?

Reply:
No, same